doc/buildTools.md

```diff
@@ -3,16 +3,31 @@
 Note: This documentation explains some commands that can be used for the development of components in Robocomp. These commands are not intended to compile Robocomp itself. To follow the necessary steps to install and compile Robocomp, you can follow the instructions in this [document](https://github.com/robocomp/robocomp#installation-from-source).
 
 Following are a list of commands and explanations for Robocomp build tools.
-## rc_init_ws
+## rcworkspace
 
-This will initialize a Robocomp workspace in the current/specified directory. If there is already a workspace (has a .rc_workspace file), then it will register it as a workspace on your computer. To de-register a workspace just delete the .rc_workspace file. 
+This tool helps in managing robocomp workspaces. 
+
+####To register a new workspace
+
+```bash
+mkdir -p ~/test_ws/src
+```
+Then add a component in the **src** directory follow [Creating my first component using robocompdsl](robocompdsl.md).
```

Comment:
I think this is not needed in here. We can just say to execute rcworkspace in the folder that contains the components?